Documentation/SUMMARY.md

```diff
@@ -0,0 +1,53 @@
+# Table of Contents
+
+* [CSI test repository welcome](csi_block_storage_kc_welcome.md)
```

Comment:
this is no longer a "test", unless you don't plan to merge this PR

Reply:
@oriyarde  I was pretty sure that I updated this last week. I'll take a look and update. thanks

Reply:
I just didn't push the commit. done.